docs/manual-deployment.md

```diff
@@ -31,10 +31,10 @@ cp devops/.env.sample devops/.env
 | `TERRAFORM_STATE_CONTAINER_NAME` | The name of the blob container to hold the Terraform state *Default value is `tfstate`.* |
 | `IMAGE_TAG` | The default tag for Docker images that will be pushed to the container registry and deployed with the Azure TRE. |
 | `ACR_NAME` | A globally unique name for the Azure Container Registry (ACR) that will be created to store deployment images. |
-| `ARM_SUBSCRIPTION_ID` | *Optional for manual deployment.* The Azure subscription ID for all resources. |
+| `ARM_SUBSCRIPTION_ID` | The Azure subscription ID for all resources. |
```

Comment:
Where is this needed - can we not get it in a script via the az cli? Might be one we can remove later.

Reply:
We should be able to get it from the current `az cli` session but we are not everywhere. Come to think of it, most places we overwrite whatever value the user provides here with `ARM_SUBSCRIPTION_ID=$(echo "$AZURE_CREDENTIALS" | jq -r '.subscriptionId')`

I've created a bug #681. I'll revert this change for now